docs/ref/validators.txt

```diff
@@ -145,6 +145,11 @@ to, or in lieu of custom ``field.clean()`` methods.
         ``['localhost']``. Other domains that don't contain a dot won't pass
         validation, so you'd need to whitelist them as necessary.
 
+    The :class:`EmailValidator` works in accordance with the HTML5 Email
+    validation standard as described `here`_.
+
+    .. _here: https://html.spec.whatwg.org/multipage/forms.html#valid-e-mail-address
```

Comment:
link "HTML5 Email validation standard" and chop "as described here".
Add a versionchanged annotation as described at https://docs.djangoproject.com/en/dev/internals/contributing/writing-documentation/#documenting-new-features.